common/djangoapps/track/views/segmentio.py

```diff
@@ -220,6 +220,10 @@ def track_segmentio_event(request):  # pylint: disable=too-many-statements
 
     context['ip'] = segment_properties.get('context', {}).get('ip', '')
 
+    # For Business Intelligence events: Add label to context
+    if 'label' in segment_properties:
```

Comment:
Nit: If it's ok for `context['label']` to be `None`, this could be simplified to `context['label'] = segment_properties.get('label')`